### DISCUSSION TITLE
babel copyFiles flag copying node_modules too to build folder

### DISCUSSION THREAD

**Discussion by Salmandabbakuti** [babel/babel#14702] [General]
babel `--copy-files` flag copying `node_modules` too to the build folder. even I tried multiple options like specifying the `ignore` path in `babel.config.json`

babel version: 7+

Here's my build command:
```
"build": "rimraf dist && babel ./ --copy-files --out-dir dist",
```

**Comment by JLHwung** [comment]
Does
```
babel ./ --copy-files --out-dir dist --ignore node_modules
```
work? The `ignore` provided by `babel-cli` will work with the `--copy-files` flag.

**Reply by Salmandabbakuti** [reply]
No. It doesn't work
